<<<<<<<< HEAD:patches/llvm/nvptx_feature_ptx60.patch
index 0000000..0000000 100644
|||||||| eb59b06:nvptx_feature_ptx60.patch
diff --git a/llvm/lib/Target/NVPTX/NVPTX.td b/llvm/lib/Target/NVPTX/NVPTX.td
index 1d947ef1ce6..eca18e125dd 100644
========
diff --git a/llvm/lib/Target/NVPTX/NVPTX.td b/llvm/lib/Target/NVPTX/NVPTX.td
index 4d4203c50376..7b50ae552227 100644
>>>>>>>> master:patches/llvm/nvptx_feature.patch
--- a/llvm/lib/Target/NVPTX/NVPTX.td
+++ b/llvm/lib/Target/NVPTX/NVPTX.td
@@ -117,16 +117,16 @@ class Proc<string Name, list<SubtargetFeature> Features>
 
 def : Proc<"sm_20", [SM20, PTX32]>;
 def : Proc<"sm_21", [SM21, PTX32]>;
-def : Proc<"sm_30", [SM30]>;
-def : Proc<"sm_32", [SM32, PTX40]>;
-def : Proc<"sm_35", [SM35, PTX32]>;
-def : Proc<"sm_37", [SM37, PTX41]>;
-def : Proc<"sm_50", [SM50, PTX40]>;
-def : Proc<"sm_52", [SM52, PTX41]>;
-def : Proc<"sm_53", [SM53, PTX42]>;
-def : Proc<"sm_60", [SM60, PTX50]>;
-def : Proc<"sm_61", [SM61, PTX50]>;
-def : Proc<"sm_62", [SM62, PTX50]>;
+def : Proc<"sm_30", [SM30, PTX60]>;
+def : Proc<"sm_32", [SM32, PTX60]>;
+def : Proc<"sm_35", [SM35, PTX60]>;
+def : Proc<"sm_37", [SM37, PTX60]>;
+def : Proc<"sm_50", [SM50, PTX60]>;
+def : Proc<"sm_52", [SM52, PTX60]>;
+def : Proc<"sm_53", [SM53, PTX60]>;
+def : Proc<"sm_60", [SM60, PTX60]>;
+def : Proc<"sm_61", [SM61, PTX60]>;
+def : Proc<"sm_62", [SM62, PTX60]>;
 def : Proc<"sm_70", [SM70, PTX60]>;
 def : Proc<"sm_72", [SM72, PTX61]>;
 def : Proc<"sm_75", [SM75, PTX63]>;
